test/evergreen/code_coverage_analysis.py

```diff
@@ -22,23 +23,84 @@ def read_timing_data(timing_data_path: str):
         print("Timing data: {} to {}, delta {}".format(start_time_secs, end_time_secs, delta_secs))
         return delta_secs
 
+# Calculate the per component coverage from the detailed files.
+def get_component_coverage_list(branch_coverage, coverage_files, outfile):
```

Comment:
Should we call it `get_branch_coverage` as the overall coverage is got in the same function?

Reply:
Fixed